samples/openthread/cli/README.rst

```diff
@@ -85,17 +100,82 @@ After building the sample and programming it to your development kit, test it by
          fe80:0:0:0:8467:5746:a29f:1196
          Done
 
-#. Optionally, if you are using more than one development kit, complete the following additional steps:
+.. _ot_cli_sample_testing_multiple:
 
-   a. Connect to another board using serial connection.
-   #. Test communication between the boards with the ``ot ping <ip_address_of_the_first_board>`` command.
-      For example:
+Testing with more boards
+------------------------
 
-      .. code-block:: console
+If you are using more than one development kit for testing the CLI sample, you can also complete the following testing procedures:
+
+.. contents::
+    :local:
+    :depth: 1
 
-         uart:~$ ot ping fdde:ad00:beef:0:3102:d00b:5cbe:a61
-         16 bytes from fdde:ad00:beef:0:3102:d00b:5cbe:a61: icmp_seq=1 hlim=64 time=22ms
+The following testing procedures assume you are using two development kits.
 
+Testing communication between boards
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To test communication between boards, complete the following steps:
+
+#. Make sure both development kits are programmed with the CLI sample.
+#. Turn on the developments kits.
+#. Set up the serial connection with both development kits.
+   For more details, see :ref:`putty`.
+#. Test communication between the boards with the ``ot ping <ip_address_of_the_first_board>`` command.
+   For example:
+
+   .. code-block:: console
+
+      uart:~$ ot ping fdde:ad00:beef:0:3102:d00b:5cbe:a61
+      16 bytes from fdde:ad00:beef:0:3102:d00b:5cbe:a61: icmp_seq=1 hlim=64 time=22ms
+
+Testing diagnostic module
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To test diagnostic commands, complete the following steps:
+
+#. Make sure both development kits are programmed with the CLI sample.
+#. Turn on the developments kits.
+#. Set up the serial connection with both development kits.
+   For more details, see :ref:`putty`.
+#. Make sure that the diagnostic module is enabled and configured with proper radio channel and transmission power by running the following commands on both devices:
+
+   .. code-block:: console
+
+      uart:~$ ot diag start
+      start diagnostics mode
+      status 0x00
+      Done
+      uart:~$ ot diag channel 11
+      set channel to 11
+      status 0x00
+      Done
+      uart:~$ ot diag power 0
+      set tx power to 0 dBm
+      status 0x00
+      Done
+
+#. Transmit a fixed number of packets with the given length from one of the devices.
+   For example, to transmit 20 packets that contain 100 B of random data, run the following command:
+
+   .. code-block:: console
+
+      uart:~$ ot diag send 20 100
+      sending 0x14 packet(s), length 0x64
+      status 0x00
+      Done
+
+#. Read the radio statistics on the other device by running the following command:
+
+   .. code-block:: console
+
+      uart:~$ ot diag stats
+      received packets: 20
+      sent packets: 0
+      first received packet: rssi=-29, lqi=255
+      last received packet: rssi=-30, lqi=255
+      Done
 
 Dependencies
```

Comment:
@LuDuda , @MarekPorwisz , we don't need to update Dependencies?

Reply:
There is no special dependencies for Diag module. It just needs to be enabled which it is by default.